tests/lib/nested.sh

```diff
@@ -282,15 +296,15 @@ get_image_name(){
     if [ "$(get_extra_snaps | wc -l)" != "0" ]; then
         SOURCE="custom"
     fi
-    echo "ubuntu-${TYPE}-${VERSION}-${SOURCE}.img"
+    echo "ubuntu-${TYPE}-${VERSION}-${SOURCE}-${NAME}.img"
 }
 
-get_image_path(){
-    echo "$WORK_DIR/image"
+is_generic_image(){
```

Comment:
Please add space after `)` and before `{`

Reply:
done